src/components/Footer.js

```diff
@@ -24,17 +36,42 @@ export function Versions({ loading, error, bblfshdVersion, webClientVersion }) {
     return <span>{error}</span>;
   }
 
+  const languageVersions = Object.keys(languages).reduce((acc, k) => {
+    const { version, name } = languages[k];
+
+    if (version === undefined) {
+      return acc;
+    }
+
+    acc.push(
+      <span key={k}>
+        {name} {version}
+        <br />
+      </span>
+    );
+
+    return acc;
+  }, []);
+
   return (
-    <span>
-      Babelfish server: {bblfshdVersion}. Web client: {webClientVersion}
-    </span>
+    <div>
+      <PaddedRight>
+        Babelfish server: {bblfshdVersion}. Web client: {webClientVersion}
+      </PaddedRight>{' '}
+      <DriversVersions data-tip="hello world">
```

Comment:
`hello world`? :)

Reply:
Fixed and rebased, we'll pretend this didn't happen :joy: